src/Carbon/Carbon.php

```diff
@@ -112,6 +112,11 @@ class Carbon extends DateTime
     const SECONDS_PER_MINUTE = 60;
 
     /**
+     * RFC7231 DateTime format.
+     */
+    const RFC7231 = 'D, d M Y H:i:s \G\M\T';
```

Comment:
Also you can simplify your format to `D, d M Y H:i:s T`; your tests will still pass.